ospf6d/ospf6_zebra.c

```diff
@@ -49,6 +49,23 @@ unsigned char conf_debug_ospf6_zebra = 0;
 /* information about zebra. */
 struct zclient *zclient = NULL;
 
+void ospf6_set_redist_vrf_bitmaps(struct ospf6 *ospf6)
+{
+	int type;
+	struct list *red_list;
+
+	for (type = 0; type < ZEBRA_ROUTE_MAX; type++) {
+		red_list = ospf6->redist[type];
+		if (!red_list)
+			continue;
+		if (IS_OSPF6_DEBUG_ZEBRA(RECV))
+			zlog_debug(
+				"%s: setting redist vrf %d bitmap for type %d",
+				__func__, ospf6->vrf_id, type);
```

Comment:
I think we should avoid registering redistribution for ZEBRA_ROUTE_OSPF and ZEBRA_ROUTE_ALL. There is a similar bug in BGP that was causing the Zebra process to to consume all system memory and finally get killed by Linux especially in scale scenario. We are in the process of fixing that from Vmware and the PR shall be out for review soon.

Reply:
Did you get a chance to address this?

Reply:
No @pguibert6WIND, I meant this is going to register for redistribution of OSPF6 routes back into OSPF6. Unless it is intended for certain purpose OSPF6 should not need to register for redistribution of OSPF6 routes itself. Same is the case for BGP. BGP does not need to register redistribution for BGP routes (at least for the same vrf/table).

Reply:
This comment needs to be addressed.

Reply:
Is this addressed? (more details here: https://github.com/FRRouting/frr/pull/7734/commits/0112e18a9fd8d0ae60f5c23384155a44d8508f8d#r544542742)